crowdkit/aggregation/classification/dawid_skene.py

```diff
@@ -267,9 +267,10 @@ def _m_step(data: annotations.LABELED_DATA, probas: annotations.TASKS_LABEL_PROB
         worker's errors probabilities matrix.
         """
         skilled_data = data.copy()
-        skilled_data['skill'] = probas.lookup(data.task, data.label)
+        skilled_data['skill'] = pd.Series(
+            [probas.loc[row.task, row.label]
+                for _, row in data.iterrows()], dtype=np.float64)
```

Comment:
Seems like it's gonna dramatically worsen the performance. `.iterrows()` is extremely slow.

Reply:
As we discussed, I checked the performance and indeed, iterrows() is very slow. In the end, I replaced lookup method with a solution recommended in [pandas documentation](https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#looking-up-values-by-index-column-labels).
Tests on relevance-2 and relevance-5 datasets showed that the performance is currently almost the same.